core-bundle/src/Search/Document.php

```diff
@@ -76,6 +81,15 @@ public function getBody(): string
         return $this->body;
     }
 
+    public function getContentCrawler(): Crawler
```

Comment:
Why did you add this method?

Reply:
To be able to fetch the content crawler? So we don't need multiple instances of the DomCrawler for everybody who wants to crawl the content.